content/modules/redisearch/redisearch-2-upgrade.md

```diff
@@ -0,0 +1,74 @@
+    1. In the Redis Enterprise web UI, go to the: **settings**
+    1. In **redis modules**, click **Add Module**.
+
+       ![upgrade_module-1](/images/rs/upgrade_module-1.png?width=1600&height=956)
+
+    1. Browse to the module package and upload it to the cluster.
+1. Create a new database with RediSearch 2.x:
+    1. [Create a database]({{< relref "/rs/administering/creating-databases/_index.md" >}}) and configure the database settings.
+    1. In the Redis Modules section, click ![Add](/images/rs/icon_add.png#no-click "Add"), select **RediSearch 2**, and click ![Save](/images/rs/icon_save.png#no-click "Save").
+    1. Click **Activate** to create the database.
+1. Migrate the data to the RediSearch 2.x database:
+    1. Download the RediSearch_Syncer package for your operating system.
+        - [RedHat Enterprise Linux 7](http://redismodules.s3.amazonaws.com/redisearch/RediSearchSyncer/RediSearchSyncer-rhel7.zip)
+        - [Ubuntu 18.04](http://redismodules.s3.amazonaws.com/redisearch/RediSearchSyncer/RediSearchSyncer-bionic.zip)
+        - [Ubuntu 16.04](http://redismodules.s3.amazonaws.com/redisearch/RediSearchSyncer/RediSearchSyncer-xenial.zip)
+        - [Ubuntu 14.04](http://redismodules.s3.amazonaws.com/redisearch/RediSearchSyncer/RediSearchSyncer-trusty.zip)
+    1. Extract the package with: `unzip <package_name>`
+    1. In the extracted directory, run the RediSearch_Syncer.py script:
+
+        ```sh
+        python RediSearch_Syncer.py -d <destination_url> -s <source_url> [-l] [--add-prefix <prefix>]
+        ```
+
+        Where:
+
+        - `destination url` - The replication URL of the RediSearch 2.x database that you see when you click on **Get Replica of source URL** in the database configuration in the web UI.
+        - `source url` - The replication URL of the RediSearch 1.x database that you see when you click on **Get Replica of source URL** in the database configuration in the web UI.
+        - `--add-prefix <prefix>` (optional) - Adds a prefix to the keys that are replicated to the new database
+
+        The script shows a table with the progress of the replication.
+        Press **F5** to see the updated status of the replication process.
+
+    1. Stop the processes that are sending requests to the source database so all of the data gets synchronized to the destination database.
+    1. When the status field is `st_in_sync` then you can press **Ctrl-C** to cancel the synchronization process.
+
+        {{< note >}}
+        You can also verify that the replication is complete in `shard-cli`. Connect to each shard in the source database and run `info replication` to see the replication offset.
+        Then connect to each shard in the destination database and make sure that the offset reported in `info replication` is the same as for the shard in the source database.
+        {{< /note >}}
```

Comment:
This is not a note, this should be a must otherwise we risk at data loss, the user should also not press Ctrl-c until it verifies this stage.

Also the replication offset should be compare to what the syncer is reporting and not to the target as describe here: 
https://docs.google.com/document/d/1nsgnBikQTmWO4dtVXtbHpJYO_R6qlTA0IgWVkZfWGx4/edit

Reply:
@MeirShpilraien Please review the last commit that resolves this.